src/Routes/Devices/UpdateDeviceModal.js

```diff
@@ -120,6 +112,24 @@ const UpdateDeviceModal = ({ updateModal, setUpdateModal, refreshTable }) => {
           {isMultiple ? `${deviceName.length} systems` : deviceName}
         </span>{' '}
         to latest version of the image linked to it.
+        {updateModal.deviceData.some(
+          (device) =>
+            device.deviceStatus !== 'updateAvailable' &&
+            device.deviceStatus !== 'error'
```

Comment:
The `Update` button in the Systems table toolbar is disabled when systems with `Error` or `Unresponsive` statuses are selected, even though you can select `Update` from the systems' row-level kebab dropdowns.

Reply:
That is true, it doesn't quite match the row-level kebab dropdowns. Currently it is more based upon the retry update popover, which doesn't have a retry button on Unresponsive systems. For this button, it is disabled when only Unresponsive devices are selected, but if any of the other devices have Update Available statuses, the button is enabled and the modal contains the alert stating that some systems don't have an update available. The button should also be enabled when a device with an Error status is selected, as long as it has an update available.

Reply:
That should be an easy change! I have to look into a few things first, but will work on that.